app/frontend/src/components/ches/MergePreviewDialog.vue

```diff
@@ -0,0 +1,185 @@
+                  :value="formatAsList(mergePreview[mergePreviewIndex].bcc)"
+                  hide-details="auto"
+                  readonly
+                  outlined
+                  dense
+                ></v-text-field>
+              </v-col>
+            </v-row>
+
+            <v-row>
+              <v-col cols="12" md="12">
+                <label>Subject</label>
+                <v-text-field
+                  v-model="mergePreview[mergePreviewIndex].subject"
+                  hide-details="auto"
+                  outlined
+                  readonly
+                  dense
+                ></v-text-field>
+              </v-col>
+            </v-row>
+
+            <v-row>
+              <v-col cols="12" md="12">
+                <label>Body</label>
+                <v-textarea
+                  v-if="mergePreview[mergePreviewIndex].bodyType === 'text'"
+                  v-model="mergePreview[mergePreviewIndex].body"
+                  hide-details="auto"
+                  readonly
+                  outlined
+                  dense
+                ></v-textarea>
+                <div
+                  v-else
+                  class="bodyPreviewDiv"
+                  v-html="mergePreview[mergePreviewIndex].body"
+                ></div>
+              </v-col>
+            </v-row>
+
+            <v-row v-if="mergePreview[mergePreviewIndex].attachments.lenghth > 0">
+              <v-col cols="12" md="12">
+                <label>Attachments</label>
+                <ul>
+                  <li v-for="item in mergePreview[mergePreviewIndex].attachments" :key="item.filename">
+                    {{ item.filename }}
+                  </li>
+                </ul>
+              </v-col>
+            </v-row>
+
+          </v-container>
+        </v-card-text>
+      </div>
+      <v-card-actions class="pb-10 justify-center">
+        <v-btn
+          color="primary"
+          @click="previewPrevious()"
+          :disabled="previousDisabled"
+        >
+          <span>Previous</span>
+        </v-btn>
+
+        <v-btn color="primary" @click="previewNext()" :disabled="nextDisabled">
+          <span>Next</span>
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+import { mapFields } from 'vuex-map-fields';
+
+export default {
+  name: 'MergePreviewDialog',
+
+  props: {
+    show: {
+      default: false,
+      type: Boolean,
+    },
+  },
+
+  data: () => ({
+    mergePreviewIndex: 0,
+  }),
+
+  computed: {
+    // get form data from vuex
+    ...mapFields('ches', ['mergePreview']),
+
+    previousDisabled: function () {
+      return this.mergePreviewIndex === 0;
+    },
+
+    nextDisabled: function () {
+      return this.mergePreviewIndex === this.mergePreview.length - 1;
+    },
+
+  },
+
+  methods: {
+    closeDialog() {
+      this.$emit('close-dialog');
+    },
+
+    formatAsList: function (arr) {
+      return arr.join(', ');
+    },
```

Comment:
The rest of the method functions are defined directly as functions instead of attribute functions. Let's be consistent about this:
```js
formatAsList(arr) {
  return arr.join(', ');
} 
```